### PR TITLE
[Docs] add azure open ai

### PR DESCRIPTION
**Pull request by Anna-Xiong**
## Description
- add azure open ai docs follow new vizro ai class
- wait for PR https://github.com/mckinsey/vizro/pull/438 to be merged

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

**Review by nadijagraca — APPROVED** [review]
LGTM. ⭐

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md [hunk 1]
```diff
@@ -61,3 +61,26 @@ To ensure a deterministic answer to our queries, we've set the temperature to 0.
         vizro_ai = VizroAI(model=llm)
         vizro_ai.plot(df, "describe the composition of gdp in continent")
         ```
+
+## Azure OpenAI models
+To set up Azure OpenAI with VizroAI, you'll need to configure the AzureOpenAI instance by specifying your deployment name and model name using LangChain. You can also set your environment variables for API configuration,
```
Comment (maxschulz-COL) [inline vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:66]:
```suggestion
To set up Azure OpenAI with VizroAI, you'll need to configure the `AzureOpenAI` instance by specifying your deployment name and model name using LangChain. You can also set your environment variables for API configuration,
```

**Review by maxschulz-COL — APPROVED** [review]
LGTM! I presume you have tested it?

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md [hunk 2]
```diff
@@ -61,3 +61,26 @@ To ensure a deterministic answer to our queries, we've set the temperature to 0.
...
+such as `OPENAI_API_TYPE`, `OPENAI_API_VERSION`, `OPENAI_API_BASE` and `OPENAI_API_KEY`.
+Authentication can be done via an API key directly or through Azure Active Directory (AAD) for enhanced security.
+For a detailed walk-through, refer to the [LangChain documentation](https://python.langchain.com/docs/integrations/llms/azure_openai/) directly.
```
Comment (stichbury) [inline vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:69]:
```suggestion
For a detailed walk-through, refer to the [LangChain documentation](https://python.langchain.com/docs/integrations/llms/azure_openai/).
```

**Review by stichbury — APPROVED** [review]
LGTM! 🏆

**Pull request merged**